src/main/webapp/app/exercises/text/assess/text-submission-assessment.component.ts

```
@@ -99,24 +106,9 @@ export class TextSubmissionAssessmentComponent extends TextAssessmentBaseCompone
-        alertService: AlertService,
-        accountService: AccountService,
-        assessmentsService: TextAssessmentService,
-        structuredGradingCriterionService: StructuredGradingCriterionService,
-        translateService: TranslateService,
-    ) {
-        super(alertService, accountService, assessmentsService, structuredGradingCriterionService);
-        translateService.get('artemisApp.textAssessment.confirmCancel').subscribe((text) => (this.cancelConfirmationText = text));
-        this.correctionRound = 0;
+    constructor() {
+        super();
+        this.translateService.get('artemisApp.textAssessment.confirmCancel').subscribe((text) => (this.cancelConfirmationText = text));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)